app/presenters/citable_attributes.rb

```diff
@@ -507,6 +511,8 @@ class TreatAsOralHistory < StandardTreatment
     def title
       interviewee = CitableAttributes::work_lookup(work, "creator", "interviewee")
       interviewer = CitableAttributes::work_lookup(work, "creator", "interviewer")
+      interviewer = [""] if interviewer == [nil]
```

Comment:
While it works, this is an unusual unfavored syntax for calling a class method. `CitableAttributes::work_lookup`. Instead, do `CitableAttributes.work_lookup`. Or, `self.class.work_lookup`. 

Also, rather than put in a special guard for `[""] if interviewer == [nil]`, I wonder if it makes more sense to pay attention to the return value of `work_lookup`?  What does it return? Always an array? But sometimes an array including `nil`?  Returning `[nil]` doesn't seem like a useful thing for `work_lookup` to return, I can't think of why you'd ever want it. Should it be returning an empty array (`[]`) instead?  

I think we can make this a lot fewer lines of code by paying attention to what makes sense to return in the first place, instead of returning something not useful and then guarding on it. Note that the empty array (`[]`) is already `blank?`.

Reply:
Very good suggestion! I think a strategically placed .compact can make this all go away... stay tuned.

Reply:
And perhaps the strategically placed compact goes in the `work_lookup` implementation (which I'm not looking at at the moment).

Reply:
Yup! Worked like a charm. Thanks for catching this.